include/cuco/detail/priority_queue_kernels.cuh

```diff
@@ -0,0 +1,1214 @@
+#pragma once
+
+#include <assert.h>
+#include <cooperative_groups.h>
+
+using namespace cooperative_groups;
+
+namespace cuco {
```

Comment:
```suggestion
namespace cuco {
namespace detail {
namespace cg = cooperative_groups;
```
All kernels should be placed in `detail` namespace.

Reply:
bec63f3